src/plugins/controls/public/options_list/components/options_list.scss

```diff
@@ -1,117 +1,121 @@
 
-    .optionsList__actionBarDivider {
-      height: $euiSize;
-      border-right: $euiBorderThin;
+    &.optionsList--filterBtnPlaceholder {
+      color: $euiTextSubduedColor;
+      font-weight: $euiFontWeightRegular;
     }
-  }
-}
 
-.optionsList__popoverTitle {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-}
+    .optionsList__filterInvalid {
+      color: $euiTextSubduedColor;
+      text-decoration: line-through;
+      margin-left: $euiSizeS;
+      font-weight: $euiFontWeightRegular;
+    }
 
-.optionsList__filterInvalid {
-  color: $euiTextSubduedColor;
-  text-decoration: line-through;
-  margin-left: $euiSizeS;
-  font-weight: $euiFontWeightRegular;
-}
+    .optionsList__existsFilter {
+      font-style: italic;
+    }
 
-.optionsList__existsFilter {
-  font-style: italic;
-}
+    .optionsList__negateLabel {
+      font-weight: bold;
+      font-size: $euiSizeM;
+      color: $euiColorDanger;
+    }
 
-.optionsList__loadMore {
-  font-style: italic;
-}
+    .euiFilterButton__notification {
+      background-color: $euiColorMediumShade;
+    }
 
-.optionsList__negateLabel {
-  font-weight: bold;
-  font-size: $euiSizeM;
-  color: $euiColorDanger;
+    .euiFilterButton__text-hasNotification {
+      flex-grow: 1;
+      justify-content: space-between;
+      width: 0;
+    }
+  }
 }
 
-.optionsList__actionBarFirstBadge {
-  margin-left: $euiSizeS;
+.optionsList--sortPopover {
+  width: $euiSizeXL * 7;
 }
 
-.optionsList-control-ignored-selection-title {
-  padding-left: $euiSizeS;
-}
+.optionsList__popover {
+  &.optionsList--smallPopover {
+    .optionsList__searchSortRow {
+      flex-direction: column;
+      align-items: stretch;
 
-.optionsList__selectionInvalid {
-  text-decoration: line-through;
-  color: $euiTextSubduedColor;
-}
+      .optionsList__sortButtonWrapper {
+        flex-grow: 1;
 
-.optionsList--filterBtnWrapper {
-  height: 100%;
-}
+        .optionsList__sortButtonPopoverAnchor {
+          width: 100%;
+        }
 
-.optionsList--filterBtn {
-  .euiFilterButton__text-hasNotification {
-    flex-grow: 1;
-    justify-content: space-between;
-    width: 0;
-  }
-  &.optionsList--filterBtnPlaceholder {
-    .euiFilterButton__textShift {
-      color: $euiTextSubduedColor;
+        .optionsList__sortButton {
+          width: 100%;
+        }
+      }
     }
   }
-}
 
-.optionsList--filterGroupSingle {
-  box-shadow: none;
-  height: 100%;
-  border-top-left-radius: 0;
-  border-bottom-left-radius: 0;
-  border-top-right-radius: $euiBorderRadius - 1px;
-  border-bottom-right-radius: $euiBorderRadius - 1px;
-}
+  .optionsList__actions {
+    padding: $euiSizeS;
+    padding-bottom: 0;
+    border-bottom: $euiBorderThin;
+    border-color: darken($euiColorLightestShade, 2%);
 
-.optionsList--filterGroup {
-  width: 100%;
-}
+    .optionsList__sortButton {
+      box-shadow: inset 0 0 0 $euiBorderWidthThin $euiFormBorderColor;
+      background-color: $euiFormBackgroundColor;
+    }
```

Comment:
because the `EuiInputPopover`  does not allow the popover width to be **larger** than the input it is attached to, made some changes to the options list popover to ensure it still renders nicely, even at its smallest size:


<div align="center"><img src="https://github.com/elastic/kibana/assets/8698078/f2124292-fcb6-4209-a77e-c0bb82169ac6" /></div><br/>